api/filters/group.py

```diff
@@ -0,0 +1,16 @@
+from django_filters import rest_framework as filters
+
+from api.models import Group
+
+
+class GroupFilter(filters.FilterSet):
+    for_coordinator_email = filters.CharFilter(
+        field_name="coordinators__personal_info__email", lookup_expr="iexact"
+    )
+    for_student_email = filters.CharFilter(
+        field_name="students__personal_info__email", lookup_expr="iexact"
+    )
```

Comment:
hm... why do we need this? No student will have any access to our dashboards, current or planned